docs/usb-image.rst

```diff
@@ -19,12 +19,17 @@ Both USB images are assembled together from these components:
 Changelog
 ~~~~~~~~~
 
-4.0.0
+4.0
 =====
 Released on `TBA`
 
 - Updated to a new platform version - `<TBD>` from upstream/release-`<TBD>`
```

Comment:
We don't know this yet?

Reply:
Not yet. The final platform is waiting for last Joyent upstream integration because of fixes in `vminfod`.